skyportal/handlers/api/group.py

```diff
@@ -90,12 +90,19 @@ def get(self, group_id=None):
                 return self.success(data=group)
             return self.error(f"Could not load group with ID {group_id}")
 
```

Comment:
These changes shouldn't be here; looks like a rebase is needed

Reply:
@acrellin I think you reviewed an old version, perhaps.  That's no longer in the diff.